### PR TITLE
release bitcoin_hashes 0.12

### PR DESCRIPTION
**Pull request by apoelstra**
It was a little tricky to bump the version number because of https://github.com/rust-bitcoin/rust-bitcoin/issues/1553. There are a couple other things I considered trying, which maybe we'll do for future releases, but I believe this works for now.

Maybe should wait for #1111.

### CODE REVIEW

**Comment by tcharding** [comment]
I'm in favour of releasing `bitcoin_hashes` soon as possible, I don't think we need to wait for #1111.

**Review by tcharding — APPROVED** [review]
ACK bfd401c96eb904d0d36fde4e6b3985499a9ec031

**Comment by tcharding** [comment]
> bitcoin_hashes: add CHANgELOG

Mad leetspeak :)

**Review by Kixunil — COMMENTED** [review]
Do we want to hold this until #1696 ? Or will we just do point release for it?

**Review by sanket1729 — APPROVED** [review]
utACK bfd401c96eb904d0d36fde4e6b3985499a9ec031. ChangeLog looks good to me, did not review whether all noteworthy changes were included.

**Comment by apoelstra** [comment]
Will wait for us to decide on #1696 before merging.

**Comment by tcharding** [comment]
I can't get #1696 today. Can we merge without it or else someone pick it up for me please?

**Comment by Kixunil** [comment]
Yeah, let's do it. I just learned today that releasing before 9 March would be really, really nice for me. :)

**Comment by apoelstra** [comment]
Yay!

**Comment by apoelstra** [comment]
Lol, dammit. We actually have to release `internals` first.

**Comment by apoelstra** [comment]
Oops, I did label the same commit `bitcoin_hashes-0.12.0`, I just forgot to push it! Done now.

**Pull request merged**
